node/src/chain_specs/manta.rs

```diff
@@ -157,32 +167,42 @@ fn manta_dev_genesis(
         sudo: manta_runtime::SudoConfig {
             key: Some(root_key),
         },
+        parachain_staking: ParachainStakingConfig {
+            candidates: invulnerables
+                .iter()
+                .cloned()
+                .map(|(account, _)| {
+                    (
+                        account,
+                        manta_runtime::staking::NORMAL_COLLATOR_MINIMUM_STAKE,
+                    )
+                })
+                .collect(),
+            delegations,
+            inflation_config: manta_runtime::staking::inflation_config::<manta_runtime::Runtime>(),
+        },
         parachain_info: manta_runtime::ParachainInfoConfig {
             parachain_id: MANTA_PARACHAIN_ID.into(),
         },
         collator_selection: manta_runtime::CollatorSelectionConfig {
```

Comment:
For brand new runtime, do we still need `collator_selection`?
I'm curious.

Reply:
Yes we do unfortunately, the pallet is still used as a nimbus filter stage
https://github.com/Manta-Network/Manta/blob/3c1e15d9100ba20e31ce8c3ccc0e2dd1054471b7/runtime/manta/src/lib.rs#L473

We do not need to give it any configuration tho